addbook.php

```diff
@@ -0,0 +1,55 @@
+<?php
+require 'functions.php';
+
+$db = new PDO('mysql:host=db;dbname=books', 'root', 'password');
```

Comment:
if you made this a function, rather than the *whole* db/query/execute you could use it in both stories.